client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -117,7 +146,18 @@ const PatternAssembler: Step = ( { navigation } ) => {
 							setFooter( null );
 						} }
 						onContinueClick={ () => {
-							// TODO
+							if ( siteSlugOrId ) {
+								const design = getDesign();
+								setSelectedDesign( design );
+
+								setPendingAction( () =>
+									setDesignOnSite( siteSlugOrId, design ).then( () =>
```

Comment:
I made D87310-code to let `theme-setup` API support `page_template`. So, I think we might need to set the `page_template` based on the selected patterns.

For example,
- If the user selects header, then we could set the `page_template` to `header-footer-only`
- If the user selects only sections, then we could set the `page_template` to `blank`

Reply:
I tested the diff with the UI, and it seems that none of the templates in Blank canvas allow us to set patterns in the header and content areas. I think it should work with `blank`, but it doesn't. 

I also think we should use `blank` as the default template because that should work for all our cases in the PA, instead of changing it based on the selected patterns. Is there any reason for that?

Reply:
I found the [issue](D87310-code#inline-318294). It was in the validation of the pattern IDs.